CONTRIBUTING.md

```diff
@@ -57,47 +30,42 @@ $ git push origin update-cool-package
 contributor-name wants to merge 1 commit into chromebrew:master from contributor-name:update-cool-package
 ```
 3. On the far right, take note of _**contributor-name:update-cool-package**_.  This is the PR contributor along with the branch they want to merge.
-4. Open a terminal, change to your cloned chromebrew directory and execute:
-```bash
-$ git remote add contributor-name git@github.com:contributor-name/chromebrew.git
-```
-5. Fetch their branches:
-```bash
-$ git fetch contributor-name
-```
-6. Checkout their branch:
+
+4. Open a terminal, execute the following to download their changes:
 ```bash
-$ git checkout update-cool-package
+$ CREW_TESTING=1 CREW_TESTING_REPO=https://github.com/contributor-name/chromebrew CREW_TESTING_RERO=update-cool-package crew update
```

Comment:
Might be good to mention replace `contributor-name` with your personal GitHub account.